src/Storage/Storage.Management/StorageAccount/RemoveAzureStorageAccountManagementPolicy.cs

```diff
@@ -15,9 +15,8 @@
 using Microsoft.Azure.Commands.Management.Storage.Models;
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
-using Microsoft.Azure.Management.Storage;
-using Microsoft.Azure.Management.Storage.Models;
 using System.Management.Automation;
+using Azure;
```

Comment:
Put system and others assemblies in groups. VS [sort-usings](https://docs.microsoft.com/en-us/visualstudio/ide/reference/sort-usings?view=vs-2022) may help.

Reply:
Done.